.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,111 @@
+name: "Bug report: Netdata"
+description: "Submit a report and help us improve our free and open-source Netdata Agent"
+title: "[Bug]: "
+labels: ["bug", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: "### Thank you for contributing to our project!"
+  - type: markdown
+    attributes:
+      value: |
+        <img src="https://img.shields.io/github/v/release/netdata/netdata.svg?label=latest%20stable"> <img src="https://img.shields.io/badge/dynamic/xml?url=https://storage.googleapis.com/netdata-nightlies/latest-version.txt&label=latest%20nightly&query=/text()">
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting, we'd appreciate it if you:
+          - Verify that your issue is not already reported on GitHub.
+          - Check if your Netdata is up to date. If not, we recommend that you updgrade first.
```

Comment:
Maybe `- Check if your Netdata installation is up to date.` ? Also `updgrade` -> `upgrade` :-)

Reply:
I think I'll go with "Netdata agent". Also, not sure which variant is _correct_: agent or Agent. I see we use both in https://learn.netdata.cloud/docs/agent/packaging/installer.

As for "updgrade", it is a combination of "update" and "upgrade".